### PR TITLE
Add Cape Verde and Somalia into countries

### PR DESCRIPTION
**Pull request by rileywaffle**
If you have any questions, msg me here or on discord @ rileywaffle

### CODE REVIEW

File: src/countries/Somalia.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	1720064, // Nugaal 
+	13316065, // Sahil 
+	1720065, // Sanaag 
+	1720062, // Shabeellaha Dhexe 
+	1720060, // Shabeellaha Hoose 
+	1720066, // Sool
+	1720067, // Togdheer	
+	1720068, // Woqooyi Galbeed
+  ];
+}
```
Comment (agatemosu) [inline src/countries/Somalia.ts]:
```suggestion
    1720059, // Jubbada Hoose
    1720063, // Mudug
    1720064, // Nugaal 
    13316065, // Sahil 
    1720065, // Sanaag 
    1720062, // Shabeellaha Dhexe 
    1720060, // Shabeellaha Hoose 
    1720066, // Sool
    1720067, // Togdheer	
    1720068, // Woqooyi Galbeed
  ];
}

```

**Comment by rileywaffle** [comment]
Also added Bolivia.

**Comment by rileywaffle** [comment]
Added 7 more counties above.

**Comment by rileywaffle** [comment]
Added 5 more counties above.

**Comment by rileywaffle** [comment]
Added 18 more, finished all WIP's that were complete.

**Comment by rileywaffle** [comment]
added a few more countries

**Comment by uzervlad** [comment]
Merging for now, but Cabo Verde division doesn't seem to match what I could find, so I'll skip it this release.

**Pull request merged**